playbooks/roles/ocp-post-install/README.md

```diff
@@ -0,0 +1,48 @@
+ocp-post-install: OCP Post install steps
+=========
+
+This module will run all the post install steps. Currently following features are supported:
+
+**1. RHCOS kernel options via MachineConfig**
+All the `rhcos_kernel_options` will be applied via MachineConfig to the master and worker nodes. The CRDs will be created at `workdir` and applied to each master and worker node. It will take some minutes to apply the configuration and nodes to be in Ready status again (depends on the number of master and worker nodes). To get the node status run the below command and ensure none of the node is in SchedulingDisabled status before you start using the cluster.
+
+```
+oc get nodes
+```
+
+Requirements
+------------
+
+ - A working OCP 4.X cluster
+ - All the nodes should be in Ready status
```

Comment:
The above is not a requirement. Instead we can say all the nodes requiring customization should be in `Ready` status